public/js/icinga/events.js

```diff
@@ -457,10 +457,16 @@
                 return true;
             }
 
+            // Check for ctrl or cmd click to open new tab unless clicking on a multiselect row
+            if ((event.ctrlKey || event.metaKey) && ! $a.parent().parent('table').length > 0 && href !== '#') {
```

Comment:
Please use the same condition as on line no. 500 to check for multiselect tables. (And outsource this to a variable for re-use) The current condition also matches the tactical overview's fake-links and still opens a new column due to this.

Reply:
If I use the multiselect condition other tables and lists (e.g. the `/icingaweb2/user/list` that you mentioned will no longer work as intended.
The tactical overview does not really fit in with the way we display in other views in general - maybe that should change?

Reply:
The condition I've mentioned also verifies that it's a `tr` tag. Links shouldn't be affected by this.

Reply:
So, what if instead i just verify that it is an anchor?
Just `(event.ctrlKey || event.metaKey) && $a.is('a') && href !== '#'`

Reply:
That's also fine. Though, then I'd rather want to see the check at the very end. The `$a.is('a')` is probably the most expensive one.